fluentui_core/build.gradle

```diff
@@ -77,6 +78,7 @@ dependencies {
     implementation "androidx.compose.ui:ui-text:$composeVersion"
     implementation "androidx.compose.ui:ui-graphics:$composeVersion"
     implementation "androidx.compose.ui:ui-unit:$composeVersion"
+    implementation "androidx.compose.ui:ui-util:$composeVersion"
```

Comment:
Shouldn't we include all compose.ui libraries? Including one lib at a time isn't the most optimised way.